### ISSUE TITLE
Informational messages are not reported to host

### ISSUE THREAD

**Issue by BernieWhite** [BernieWhite/PSRule#48]
**Description of the issue**

Informational messages are not reported to host under the following conditions:

- Verbose, information, warning and error messages within rule definition blocks
- General verbose, information and warning messages within `.Rule.ps1` scripts outside of rule definitions

**To Reproduce**

Steps to reproduce the issue:

```powershell
# Logging in main script
Write-Information -MessageData 'Script information message';
Write-Verbose -Message 'Script verbose message';
Write-Warning -Message 'Script warning message';

Rule 'WithError' {
    Write-Error -Message 'Rule error message';
    $True;
}

Rule 'WithWarning' {
    Write-Warning -Message 'Rule warning message';
    $True;
}

Rule 'WithVerbose' {
    Write-Verbose -Message 'Rule verbose message';
    $True;
}

Rule 'WithInformation' {
    Write-Information -MessageData 'Rule information message';
    $True;
}
```

```powershell
Invoke-PSRule -Verbose -InformationAction Continue;
```

**Expected behaviour**

Verbose, information, warning and error messages should be returned to the host PowerShell runspace where they can interpreted or saved.

**Module in use and version:**

- Module: PSRule
- Version: 0.1.0
